src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -69,9 +72,9 @@ export class SavedObjectsClient {
     const {
       overwrite = false
     } = options;
-
     const objectToBulkRequest = (object) => {
       const method = object.id && !overwrite ? 'create' : 'index';
+      const time = this._getCurrentTime();
```

Comment:
Actually, we're not returning the `created_at` in the objects for this function and we should be. So will most likely require re-using that assignment.